truffleruby/src/main/java/org/truffleruby/Main.java

```diff
@@ -139,19 +139,15 @@ public static void main(String[] args) {
                         Log.performanceOnce("this JVM does not have the Graal compiler - performance will be limited - see doc/user/using-graalvm.md");
                     }
 
+                    final String bootCode;
                     if (config.shouldUsePathScript()) {
                         context.setOriginalInputFile(config.getScriptFileName());
-                        exitCode = engine.eval(loadBootSource(
-                                //language=ruby
```

Comment:
Please do not remove these comments, intellij is able to index to Ruby code with them in the Strings.